code/game/machinery/computer/cloning.dm

```diff
@@ -526,7 +526,11 @@
 			has_bank_account = I.registered_account
 	if(isbrain(mob_occupant))
 		dna = B.stored_dna
-
+	
+	if(HAS_TRAIT(mob_occupant, NO_DNA_COPY))
+		scantemp = "<font class='bad'>Silicon-Based lifeforms do not have DNA and therefor cannot be cloned!</font>"

```

Comment:
This is still too specific, make it something along the lines of "The DNA of this lifeform could not be read" or some bs like that